source/tests/utilities/client_helpers.h

```diff
@@ -49,7 +49,7 @@ class simple_variant {
 inline void raise_if_error(const ::grpc::Status& status)
 {
   if (!status.ok()) {
-    throw new std::runtime_error(status.error_message());
+    throw std::runtime_error(status.error_message());
```

Comment:
Oops!

Reply:
uhmm.. where is this used? Or is it something indirectly used by new client generator pattern that you are using in dcpower?

Reply:
Yes, that's used by the client generator. The generated tests throw on bad status so they can just return a response. I'll probably add a more specific exception class at some point, but not very many tests care about specific bad statuses.